contracts/upgradeable_contracts/arbitrary_message/BasicAMB.sol

```diff
@@ -47,27 +52,27 @@ contract BasicAMB is BasicBridge {
     }
 
     function setSubsidizedModeForHomeToForeign() external onlyOwner {
-        bytesStorage[HOME_TO_FOREIGN_MODE] = SUBSIDIZED_MODE;
+        uintStorage[HOME_TO_FOREIGN_MODE] = SUBSIDIZED_MODE;
     }
 
     function setDefrayalModeForHomeToForeign() external onlyOwner {
-        bytesStorage[HOME_TO_FOREIGN_MODE] = DEFRAYAL_MODE;
+        uintStorage[HOME_TO_FOREIGN_MODE] = DEFRAYAL_MODE;
     }
 
     function setSubsidizedModeForForeignToHome() external onlyOwner {
-        bytesStorage[FOREIGN_TO_HOME_MODE] = SUBSIDIZED_MODE;
+        uintStorage[FOREIGN_TO_HOME_MODE] = SUBSIDIZED_MODE;
```

Comment:
does it make sense to raise the event `ForeignToHomeModeChanged` here?

Reply:
Added events for the setters https://github.com/poanetwork/poa-bridge-contracts/pull/77/commits/cd236e08095fd254a32e661bde88a86d394824f1